app/pages/nonsipp/BasicDetailsCompletedPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp
+
+import pages.QuestionPage
+import models.SchemeId.Srn
+import play.api.libs.json.{__, JsPath}
+import viewmodels.models.SectionCompleted
+
+case class BasicDetailsCompletedPage(srn: Srn) extends QuestionPage[SectionCompleted] {
```

Comment:
Do we need to set this value up in FE mongo cache on GET from ETMP?

Reply:
No, since we don't go back to amend basic details after a submission theres no need